airflow/models/dagrun.py

```diff
@@ -125,14 +125,16 @@ def refresh_from_db(self, session=None):
 
     @staticmethod
     @provide_session
-    def find(dag_id=None, run_id=None, execution_date=None,
+    def find(dag_id=None, dag_ids=None, run_id=None, execution_date=None,
```

Comment:
Similar to the comment of @kaxil and @ashb earlier, I think we should combine the `dag_id` and `dag_ids` 👍

Reply:
Can we add type annotations as well? :)

Reply:
I will combine dag_id and dag_ids parameter.

Here are only primitive types, so I'll add types annotations.